packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -286,8 +301,12 @@ export async function sideloadAddIn(manifestPath: string, app?: OfficeApp, canPr
 
   if (isDesktop && app) {
     if (isSideloadingSupportedForDesktopHost(app)) {
-      await registerAddIn(manifestPath);
-      sideloadFile = await generateSideloadFile(app, manifest, document);
+      if (app == OfficeApp.Outlook) {
```

Comment:
It looks like you could write the dev settings here just before you get the Outlook exe path and launch it.

Reply:
I also think you don't really need to check whether or not it is already in the registry. It can just write the registry value each time and it is OK if it is writing a previous value (as you can only sideload one add-in at a time).

Reply:
I think this may make it more difficult to test but I will have to see

Reply:
Once you see that the app is outlook, it could call to a separate function for doing the outlook sideloading.

Reply:
I followed your guidance here and moved everything, including enableOutlookSideloading, into a separate sideloadOutlookAddin function

Reply:
Thanks. Did you push a new commit? I don't see one.

Reply:
No not yet - wrapping up remaining comments

Reply:
When you push new commits, it doesn't tell others, so it can help to provide a message when you want the additional changes review.